Makefile

```diff
@@ -129,9 +154,29 @@ cluster/clean:
 	- kubectl delete -f deploy/role.yaml
 	- kubectl delete -n $(NAMESPACE) -f deploy/role_binding.yaml
 	- kubectl delete -n $(NAMESPACE) -f deploy/service_account.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
 	- make monitoring/uninstall
-	- kubectl delete namespace $(NAMESPACE)
\ No newline at end of file
+	- kubectl delete namespace $(NAMESPACE)
+	- kubectl delete namespace ${APP_NAMESPACES}
+
+
+.PHONY: image/build/master
+image/build/master:
+	operator-sdk build quay.io/${ORG_NAME}/${APP_NAME}:master
+
+ .PHONY: image/push/master
+image/push/master:
+	docker push quay.io/${ORG_NAME}/${APP_NAME}:master
+
+
+
+ .PHONY: image/build/dev
+image/build/dev:
+	operator-sdk build quay.io/${ORG_NAME}/${APP_NAME}:${DEV_TAG}
```

Comment:
Is `${DEV_TAG}` set at this point? I can't see it. Maybe something like `DEV_TAG ?= $(shell sh -c "git rev-parse --short HEAD")` at the top of the file is needed?

Also given that there will not be an `image/build/master` and `image/push/master`, maybe the `/dev` can be removed from these task names. Wdyt?